### PR TITLE
Wura corpus class file

### PR DESCRIPTION
**Pull request by lflage**
Added the wura dataset corpus class file

### CODE REVIEW

File: src/lm_datasets/datasets/multilingual/wura.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    )
+
+    HOMEPAGE = "https://huggingface.co/datasets/castorini/wura"
+    LICENSE = License(
+        name="Apache License Version 2.0",
+        url="http://www.apache.org/licenses/LICENSE-2.0"
+    )
+    AVAILIBILITY = Availability.DIRECT_DOWNLOAD
+
+    LANGUAGES = ["en", "fr", "pt", "af", "am", "ha", "ig", "rw", "mg", "ny",
```
Comment (malteos) [inline src/lm_datasets/datasets/multilingual/wura.py:23]: Can you create for each language a new dataset class? So you create one "base" clasas and then each language subclass inherits from it but you only change the dataset_id and the languages. Essentially like here https://github.com/malteos/lm-datasets/blob/main/src/lm_datasets/datasets/multilingual/macocu.py

File: src/lm_datasets/datasets/multilingual/wura.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+from lm_datasets.datasets.base import Availability, GB, License
+from lm_datasets.datasets.hf_dataset import HFDataset
+
+class WuraCorpus(HFDataset):
+    """
+    OVERLAP_WITH mc4
+    """
+    DATASET_ID = "wura"
```
Comment (malteos) [inline src/lm_datasets/datasets/multilingual/wura.py:8]: If you make the language subclass, please also add the `SOURCE_ID = "wura"`

File: src/lm_datasets/datasets/multilingual/wura.py (hunk 1 above)
Comment (lflage) [reply src/lm_datasets/datasets/multilingual/wura.py:23]: Should add them separeatly to the registry?
Or only the base Wura class?

Comment (malteos) [reply src/lm_datasets/datasets/multilingual/wura.py]: No please add all subclasses but not the base class.

**Comment by malteos** [comment]
You only need to add the subclasses to the rehistry and then we can merge it.

**Pull request merged**